### ISSUE TITLE
Prepare torch model for MNIST classification

### ISSUE THREAD

**Issue by marmarmarmar** [quczer/latent-geometry#2]
Find VAE (or autoencoder) trained on MNIST (implemented in pytorch).

Ideally - it should be trained with MSE and have 2-dimensional latent.

If it is not trained with MSE - that's ok.
If the latent is not 2-dim:

1. Embed MNIST Test into latent,
2. Train 2-dim PCA on these representations,
3. Extend your model with this PCA.

x -> ENC -> latent -> DEC -> x'
x -> ENC -> PCA -> latent2d -> PCA^(-1) -> DEC -> x'
